frontend/public/components/namespace.jsx

```diff
@@ -124,149 +128,376 @@ const fetchNamespaceMetrics = () => {
+  metrics: UI.getIn(['metrics', 'namespace']),
+  selectedColumns: UI.getIn(['columnManagement']),
+});
+
+const NamespacesTableRow = connect(namespacesRowStateToProps)(
+  ({ obj: ns, index, key, style, metrics, selectedColumns }) => {
+    const requester = getRequester(ns);
+    const bytes = _.get(metrics, ['memory', ns.metadata.name]);
+    const cores = _.get(metrics, ['cpu', ns.metadata.name]);
+    const description = getDescription(ns);
+    const labels = ns.metadata.labels;
+    const columns = new Set(
+      selectedColumns?.get(NamespacesColumnManagementID) || getNamespacesSelectedColumns(),
+    );
+    return (
+      <TableRow id={ns.metadata.uid} index={index} trKey={key} style={style}>
+        <TableData className={projectAndNamespaceColumnInfo.name.classes}>
+          <ResourceLink kind="Namespace" name={ns.metadata.name} title={ns.metadata.uid} />
+        </TableData>
+        <TableData
+          className={projectAndNamespaceColumnInfo.displayname.classes}
+          columns={columns}
+          columnID={projectAndNamespaceColumnInfo.displayname.id}
+        >
+          <span className="co-break-word co-line-clamp">
+            {getDisplayName(ns) || <span className="text-muted">No display name</span>}
+          </span>
+        </TableData>
+        <TableData
+          className={classNames(projectAndNamespaceColumnInfo.status.classes, 'co-break-word')}
+          columns={columns}
+          columnID={projectAndNamespaceColumnInfo.status.id}
+        >
+          <Status status={ns.status.phase} />
+        </TableData>
+        <TableData
+          className={classNames(projectAndNamespaceColumnInfo.requester.classes, 'co-break-word')}
+          columns={columns}
+          columnID={projectAndNamespaceColumnInfo.requester.id}
+        >
+          {requester || <span className="text-muted">No requester</span>}
+        </TableData>
+        <TableData
+          className={projectAndNamespaceColumnInfo.memory.classes}
+          columns={columns}
+          columnID={projectAndNamespaceColumnInfo.memory.id}
+        >
+          {bytes ? `${formatBytesAsMiB(bytes)} MiB` : '-'}
+        </TableData>
+        <TableData
+          className={projectAndNamespaceColumnInfo.cpu.classes}
+          columns={columns}
+          columnID={projectAndNamespaceColumnInfo.cpu.id}
+        >
+          {cores ? `${formatCores(cores)} cores` : '-'}
+        </TableData>
+        <TableData
+          className={projectAndNamespaceColumnInfo.created.classes}
+          columns={columns}
+          columnID={projectAndNamespaceColumnInfo.created.id}
+        >
+          <Timestamp timestamp={ns.metadata.creationTimestamp} />
+        </TableData>
+        <TableData
+          className={projectAndNamespaceColumnInfo.description.classes}
+          columns={columns}
+          columnID={projectAndNamespaceColumnInfo.description.id}
+        >
+          {description || <span className="text-muted">No description</span>}
```

Comment:
```suggestion
          <span className="co-break-word co-line-clamp">
            {description || <span className="text-muted">No description</span>}
          </span>
```

Reply:
Done.

Reply:
Looks like line clamp is missing?

Reply:
Sorry, I only fixed this in `ProjectTableRow` but forget the same one in `NamespacesTableRow`, now fixed.